plugin-developer-learning-pathway/07-Custom-Database-Tables/01-Creating-and-Managing-Custom-Tables.md

```diff
@@ -16,54 +18,76 @@ Even still, there are times when creating a custom table can be more effective.
 
 Now that you can identify when custom database tables are useful, let's learn how to add one!
 
-## The `wpdb` Class in PHP
+In this example, you're going to create a custom table to store form submissions, which will have a field for the name and a message. 
+
+## The `WPDB` class
```

Comment:
If I'm not mistaken, the class is indeed in all lowercase per the class's definition in WordPress, seen here: https://github.com/WordPress/WordPress/blob/324f713adf6564be91349d6c907229cea3fab22a/wp-includes/class-wpdb.php#L55

Reply:
Whoops, you are 100% correct. I'm so used to most other classes not being all lower case. I will amend.